src/universal/modules/teamDashboard/components/Team/Team.js

```diff
@@ -122,9 +132,10 @@ const styleThunk = () => ({
 });
 
 export default createFragmentContainer(
-  withRouter(withStyles(styleThunk)(Team)),
+  withAtmosphere(withRouter(withStyles(styleThunk)(Team))),
   graphql`
     fragment Team_team on Team {
+      contentFilter
```

Comment:
with redux, we'd need a "teamReducer", holding it all in one place is really nice, no boilerplate + a schema so we get a self-documenting API